NuCivic/dkan
# Update chosen config exclusions

## Description

Group node search filters should not use the chosen widget 
![advisory_council_for_infectious_disease___dkan](https://cloud.githubusercontent.com/assets/314172/25713352/08e9699e-30ba-11e7-942e-d7078f31581d.png)

## QA Steps
Visit a group node and confirm that the filter options are not using the chosen widget


## Reminders
